basic/src/main/resources/application-prod.properties

```diff
@@ -1,3 +1,5 @@
 #Do not edit, version properties are set at build time
 c8y.version=@c8y.version@
-microservice.version=@project.version@
\ No newline at end of file
+microservice.version=@project.version@
+
+management.endpoints.web.exposure.include=health, loggers
```

Comment:
before change this, I had a problem when deploying the microservice: its state never got green/active in the UI. I suspect it was b/c of the missing exposed health endpoint - so the question is: should I add the health endpoint here (and in all other properties files?

Reply:
mhhm, i deployed the micoroservice without managment.endpoints on tenant 1018.0.174. The template_basic microservice got green and also the health endpoint is available.